assets/css/_vehicle_properties_panel.scss

```diff
@@ -37,6 +37,34 @@
   @include font-header;
 }
 
+.m-vehicle-properties-panel__schedule-adherence {
+  @include font-small;
+  color: $color-font-dark;
+  display: flex;
+  margin-top: 0.7rem;
+}
+
+.m-vehicle-properties-panel__schedule-adherence-status-icon {
+  margin-right: 0.4rem;
+  stroke: none;
+
+  &.on-time {
+    fill: $color-vehicle-ontime;
+  }
+  &.early {
+    fill: $color-vehicle-early;
+  }
+  &.late {
+    fill: $color-vehicle-late;
+  }
+}
+
+.m-vehicle-properties-panel__schedule-adherence-status-string {
+  font-weight: bold;
+  margin-right: 0.3rem;
+  text-transform: uppercase;
```

Comment:
Could use `@include font-emph` here to avoid rewriting these text styling properties and stay consistent with the ui kit.  It would end up being 14px instead of 12px (the label would also need to change to `font-body`).

Here's how it'd look using `@include font-emph`.  @ingridpierre thoughts? 
<img width="384" alt="Screen Shot 2019-06-10 at 16 47 34" src="https://user-images.githubusercontent.com/23065557/59225751-87728100-8b9f-11e9-96ae-89d38272d62b.png">

Reply:
After talking in person, it should stay small.